ccd-client/src/main/java/uk/gov/hmcts/cmc/ccd/client/service/SubmitCase.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.cmc.ccd.client.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.cmc.ccd.client.header.HttpHeadersFactory;
+import uk.gov.hmcts.cmc.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.cmc.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.cmc.ccd.client.model.EventRequestData;
+
+@Service
+public class SubmitCase {
+
+    private final RestTemplate restTemplate;
+    private final HttpHeadersFactory headersFactory;
+
+    @Value("${ccd.caseDataStore.baseUrl}")
+    private String ccdUrl;
```

Comment:
Please don't use field injection, it makes testing harder.

Reply:
Plus we use constructor injection everywhere, we should be consistent.

Reply:
done